server/selinux/server/relabel.sh

```diff
@@ -1,7 +1,18 @@
 #!/usr/bin/env bash
 
+function version_less_than {
+# Determines if the version passed in as the first argument is less than the version in the second
+# argument.
+    if [[ $(echo -e $1'\n'$2|sort -V|head -n 1) != $2 ]]
+    then
+        return 0
+    else
+        return 1
+    fi
+}
```

Comment:
bash does the implicit return thing, sorta like ruby, so this can just be
```bash
function version_less_than {
# Determines if the version passed in as the first argument is less than the version in the second
# argument.
    [[ $(echo -e $1'\n'$2|sort -V|head -n 1) != $2 ]]
}
```